src/main/java/com/Bootcamp/TwitterBasic/BasicTwitterFeatures.java

```diff
@@ -0,0 +1,49 @@
+package com.Bootcamp.TwitterBasic;
+
+import twitter4j.Status;
+import twitter4j.Twitter;
+import twitter4j.TwitterException;
+import twitter4j.TwitterFactory;
+
+import java.util.List;
+import java.util.Scanner;
+
+public class BasicTwitterFeatures {
+    public void postTweet(String text)
+    {
+        Twitter twitter = TwitterFactory.getSingleton();
```

Comment:
While it is a singleton, it is alright to call it any number of times for each function, just make sure TwitterrFactory.getSingleton() isn't time or memory consuming.

Also good time to read and know about Singleton design pattern.